ci-operator/jobs/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main-presubmits.yaml

```diff
@@ -0,0 +1,72 @@
+presubmits:
+  konflux-workspaces/workspaces:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
```

Comment:
is this needed?

```suggestion
```

Reply:
Most of this file is autogenerated from the config above, and taking this out has it get replaced the next time it's refreshed.